src/mail/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "html-to-text": "8.1.0",
+    "mjml": "4.11.0"
+  },
+  "scripts": {
+    "build-mjml-to-html": "mjml mjml/*.mjml -o ../backend/marsha/core/templates/core/mail/html/;",
+    "build-mails-html-to-plain-text": "for file in ../backend/marsha/core/templates/core/mail/html/*.html; do html-to-text < $file > ${file%.html}.txt; done; mv ../backend/marsha/core/templates/core/mail/html/*.txt ../backend/marsha/core/templates/core/mail/text/; ",
+    "build-mails": "yarn build-mjml-to-html; yarn build-mails-html-to-plain-text;"
```

Comment:
I don't know if we should keep it like this or create a js file executed by node. The js file will be easier to read, understand and debug.

Reply:
this one is quiet huge as well in another package.json, the path is long so it seems that the instruction is huge but it only parse the folder, I wait for your final opinion on this one.
"extract-translations": "formatjs extract './**/*.ts*' --ignore ./node_modules --ignore './**/*.d.ts' --out-file i18n/frontend.json --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format crowdin",

Reply:
Yes the extract translations is long but, as I know, we don't have alternative. And the usage is not so hard, we are just using available options. These options are documented and easy to understand.
In the `build-mails-html-to-plain-text` you are writing bash instructions, a loop, redirecting input and output. And all of this in a one-line "program".
At least doing this in shell file will be easier to understand. And in this case you don't have all the complexity to write a node program working with the filesystem. I think It's a good solution.

Reply:
yes shell is a good one!